template/src/components/routes/RouteFromPath.js

```diff
@@ -0,0 +1,17 @@
+import { Route } from 'react-router-dom';
+import { node } from 'prop-types';
+
+import PrivateRoute from './PrivateRoute';
+
+const RouteFromPath = ({ component, ...route }) =>
+  route.private ? (
```

Comment:
Can `private` flag can be destructured here? There's no need to pass it to child components right?
```suggestion
const RouteFromPath = ({ component, private, ...route }) =>
  private ? (
```

Reply:
`private` is a reserved word in js, we can't use it in this case

Reply:
oh wow, that's fine then 😂